### DISCUSSION TITLE
Progress Report 20–24 June

### DISCUSSION THREAD

**Discussion by esheehan-gsl** [NOAA-GSL/unified-graphics#41] [Progress Report]
🚴🏻 **Progress**

- #38 
- Finalized agenda for demo
- Wireframed the concept for the next iteration of the UI

😖 **Regressions**

None

💬 **To Discuss**

- Next steps for the application
- Uploading new diag files from model runs

📆 **Next Week**

- Demo
- Multiple variable display in the application

**Comment by ian-noaa** [comment]
🚴🏻 **Progress**

- `main` images are now deployed in the cluster
- The `ui` image can be accessed from an IP

😖 **Regressions**

- None

💬 **To Discuss**

- None

📆 **Next Week**

- Work with VLab to automatically deploy `main` images into the cluster
- Work with VLab to expose the `ui` service via a URL
- Set up networking inside the cluster so the containers can communicate
- Work on infrastructure for integration tests
